main.go

```diff
@@ -75,10 +77,12 @@ func init() {
 	klog.InitFlags(nil)
 
 	_ = clientgoscheme.AddToScheme(scheme)
-	_ = clusterv1alpha2.AddToScheme(scheme)
 	_ = clusterv1alpha3.AddToScheme(scheme)
+	_ = clusterv1alpha4.AddToScheme(scheme)
```

Comment:
Same as before - just v1alpha4?

Reply:
The existing file had imports to both v1a2 and v1a3 packages, hence followed suite.

More of a generic question, what is the scheme used for? FWIU, this is used to register types in a particular version. Wouldn't then the v1a3 types be needed for upgrade purposes?

Reply:
The scheme allows us to encode/decode specific GroupVersionKinds (e.g. a v1alpha3 Cluster or a v1alpha4 Cluster). If all our controllers are only talking v1alpha4, there shouldn't be any need to add v1alpha3 to the scheme.

Reply:
And since we registered conversions for v1a3 to v1a4 using the h-and-s model, the controllers would only ever use v1a4 types. Is that the correct assumption?